### ISSUE TITLE
Engine -> Framework roll blocked on new Android tracing library

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#92801]
Starting with the roll here: https://github.com/flutter/flutter/pull/92735

It looks like this PR is causing problems for the Engine -> Framework roll: https://github.com/flutter/engine/pull/29230

Many framework presubmit checks are failing with the error:

```
[        ] Execution failed for task ':app:checkDebugAarMetadata'.
[        ] > Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
[        ]    > Resolved 'androidx.tracing:tracing:1.0.0' which is not part of the dependency lock state
```

/cc @blasten @xster @dnfield 

Unfortunately https://github.com/flutter/engine/pull/29230 does not revert cleanly. Please unblock the roll to the framework ASAP because we will need to start hash alignment for the November beta on Monday morning.

**Comment by dnfield** [comment]
@blasten do we just need to update the lock files in the framework?

**Comment by jiahaog** [comment]
In case we are unable to proceed with #92800 in time, I have prepared flutter/engine#29434 as well which is a clean revert.

**Comment by zanderso** [comment]
I think if https://github.com/flutter/engine/pull/29230 will break end-users, then we should revert and create a migration plan first. If it is not a breaking change, then we should land https://github.com/flutter/flutter/pull/92800.

**Comment by zanderso** [comment]
@blasten Looks like a lock file might have been missed: https://ci.chromium.org/ui/p/flutter/builders/prod/Linux_android%20new_gallery__crane_perf/835/overview

**Comment by zanderso** [comment]
Also https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_android%20new_gallery__transition_perf/2943/overview

**Comment by cbracken** [comment]
The engine roll has now landed.
